### ISSUE TITLE
Open Chat

### ISSUE THREAD

**Issue by Adalaen** [abadias618/RoninJack#1]

**Comment by Avila3D** [comment]
Hello World

**Comment by drjev2** [comment]
hi globe

**Comment by abadias618** [comment]
@Avila3D look at this link for face recon
https://azure.microsoft.com/en-us/services/cognitive-services/face/#verification

**Comment by ChaosMartel** [comment]
Support has ended for this, but it has a good review from a month ago.  What do you all think about it? https://assetstore.unity.com/packages/tools/integration/opencv-plus-unity-85928

**Comment by herayang** [comment]
**Retrospective :** 
TL DR; 
- Use github more often.
- Collaborate more with your teammates. 
- Need AutoRun. (and maybe other features in later sprints) 
- We did 80~90% of what we planned for Sprint 1. 
- Be specific and have measurable goals. 
- Attendance matters. 

**- Make use of GitHub:** 
We have individually Working individually for too long makes it harder to combine the codes. This is why we're using GitHub, so having GitHub Desktop would be very useful. Get used to Pull/Push! 
**- Achieved what we planned for Sprint 1:**
We have achieved most of what we've decided to work on. (Face Recognition research, voice recognition, stage 1 design.) There will be some that will be carried on to Sprint 2, but we have done pretty well on our planning. Since we're more familiar with the development environment, I think we can try more for the next sprint. We now have an experience on what we can achieve, so let's plan accordingly.
**- Team effort (Needs more communication) :** 
Team suggested that there hasn't been a lot of communications between teammates, and there wasn't a lot of collaborations. We're strong, independent people who don't need nobody, but sometimes it's good to get some help. If you are struggling, don't struggle alone! Use OpenChat on Github or use Canvas Groups to communicate with your teammates. 

**-Gameplay:** 
Now that we have basic game parts getting ready, some suggested that we need to be more specific on how the game's going to be. This includes character movements (jump, left, right, rolling), different features (animals, people). Should be further dicsussed/developed in later Sprints. Right now, we need auto run. 
**IF YOU HAVE ANY GOOD IDEAS, GO TO GAMEPLAY ISSUE AND ADD YOUR SUGGESTION.**

**-(For Sprints) Practice Scrum**
We should be more specific on what we're going to do, in a measurable sense. Have a clear vision as a group, and hopefully have reminders from Scrum masters on those milestones. Have a clear understanding on what should be done by each individual. 

**-Attendance**
Let's try to be in class. Your teammates might suffer when you're not here, so try to be here on time (plus it'll hurt your grade pretty bad). Try to catch up before the class, and let people know if you have to leave early.

**Comment by Adalaen** [comment]
Ivan Regino
Antonio Avila
Timothy Gates
Jordan Evans
Hera Yang
alberto abdias 
Hunter Angelovic

**Comment by Avila3D** [comment]
Hey team, I have put the tasks for sprint 2 on Git Hub under projects as well as who is assigned to them. If you get stuck on something or need help let me know. My goal is to keep you guys in a groove so we can get more accomplished this sprint.

**Comment by Adalaen** [comment]
Link to app.
https://mega.nz/#!DkYzWAhL!cVRaLj6FN0p_h9ZonrTgRPOWUOvxLKIU7NEB0CigoQk
